build.gradle

```diff
@@ -1555,8 +1555,12 @@ project(':storage') {
 
     testImplementation project(':clients')
     testImplementation project(':clients').sourceSets.test.output
+    testImplementation project(':core')
```

Comment:
Changed `testCompile` to `testImplementation`, because `testCompile` was deprecated, and is now removed in Gradle 7.0.